docs/excel/make-your-excel-add-in-backwards-compatible-with-com-add-in-or-xll.md

```diff
@@ -0,0 +1,112 @@
+       <ProgID>{progid}</ProgID>  
+       <Type>COM</Type>  
+  </EquivalentAddin>  
+  
+  <EquivalentAddin>  
+       <FileName>{filename}.xll</FileName>  
+       <Type>XLL</Type>  
+  </EquivalentAddin>  
+<EquivalentAddins>
+...
+</VersionOverrides>
+```
+
+> [!NOTE]
+> If an add-in declares its custom functions to be XLL compatible, changing the manifest at a later time could break a user’s workbook because it will change the file format.
+
+## How the compatibility behavior works for users
+
+When an equivalent COM add-in is specified, Excel will suppress your web add-in UI on Windows when the specified COM add-in is installed. This does not affect your web add-in UI on other platforms like online or macOS. The following scenarios describe what happens depending on how the user acquires the web add-in.
+
+### Office Store acquisition  
+
+If a user downloads the web add-in from the Office store Excel will:
+
+1. Install the web add-in.
+2. Hide the web add-in UI in the ribbon.
+3. Display a call-out for the user that points out the COM add-in ribbon button.
+
+### Centralized deployment
+
+If an admin deploys the web add-in to their tenant using centralized deployment, then the user needs to restart Excel before they will see any changes.
+
+After they restart Excel, Excel will:
+
+1. Install the web add-in.
+2. Hide the web add-in UI in the ribbon.
+3. Display a call-out for the user that points out the COM add-in ribbon button.
+
+### Document share
+
+If a user has the COM add-in installed, and then gets a shared document with the ebedded web add-in, then when they open the document:
+
+1. The user will see a trust prompt for the web add-in.
+2. If trusted, the web add-in will install.
+3. Excel hides the web add-in UI in the ribbon.
+
+### User uninstalls
+
+If a user uninstalls the COM add-in, Excel restores the web add-in UI on Windows if the web add-in was previously installed.
+
+Excel only hides the ribbon buttons and does not prevent installation. Therefore your web add-in will still appear in the following UI locations:
+
+- Under **My Add-ins** because it is technically installed.
+- As an entry in the ribbon manager.
+
+### Web add-in updates
+
+Once you specify an equivalent COM addin or XLL for your web add-in, Excel stops processing updates for your web add-in. In order to get the latest updates for the web add-in, the user must uninstall the COM add-in or XLL.
+
+## Custom function behavior for XLL compatible function
+
+When a spreadsheet is opened and contains XLL functions for which there is also an equivalent add-in, the XLL functions are converted to XLL compatible custom functions. They will be saved in the file in a compatible mode such that they will still work with the original XLL, or with custom functions (when on other platforms like online or macOS).
+
+The following table compares features across XLL user defined functions, XLL compatible custom functions, and Office web add-in custom functions.
```

Comment:
Not sure. but:
"user defined" -> "user-defined"
"XLL compatible" -> "XLL-compatible"